AWSPinpoint/AWSPinpointEventRecorder.m

```diff
@@ -1149,4 +1268,20 @@ - (AWSPinpointTargetingPutEventsRequest*) putEventsRequestForEvents:(NSDictionar
     return putEventRequest;
 }
 
++ (NSMutableDictionary *)getMutableDictionaryFromResultSet:(AWSFMResultSet *)rs
+                                             forColumnName:(NSString *)columnName
+                                                     error:(NSError *__autoreleasing *)error {
+    NSSet *allowableClasses = [[NSSet alloc] initWithObjects:[NSMutableString class],
+                               [NSDictionary class],
+                               nil];
+    NSDictionary *immutableDict = [AWSNSCodingUtilities versionSafeUnarchivedObjectOfClasses:allowableClasses
+                                                                                    fromData:[rs dataForColumn:columnName]
+                                                                                       error:error];
+    if (*error) {
+        return nil;
+    }
+    NSMutableDictionary *mutableDict = [immutableDict mutableCopy];
+    return mutableDict;
```

Comment:
just `return [immutableDict mutableCopy];`

Reply:
I like declaring intervening variables to make it easier to debug.